client/my-sites/stats/stats-followers/index.jsx

```diff
@@ -66,11 +66,11 @@ class StatModuleFollowers extends Component {
 		const options = [
 			{
 				value: 'wpcom-followers',
-				label: this.props.translate( 'WordPress.com Followers' ),
+				label: this.props.translate( 'WordPress.com followers' ),
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **58** times:
`translate( 'Total WordPress.com Followers' )` *ES Score: 9*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/card-headings-form-labels-sentence-case)